plugins/obs-qsv11/CMakeLists.txt

```diff
@@ -105,17 +51,11 @@ if(OS_WINDOWS)
             device_directx9.cpp
             device_directx9.h)
 
-  target_link_libraries(obs-qsv11 PRIVATE OBS::libmfx d3d9 d3d11 dxva2 dxgi dxguid)
+  target_link_libraries(obs-qsv11 PRIVATE vpl::vpl d3d9 d3d11 dxva2 dxgi dxguid)
 elseif(OS_LINUX)
   find_package(VPL REQUIRED)
```

Comment:
And this find_package can be deleted now that vpl is being found for both platforms above.

Reply:
This change was made.